bindata/scripts/bf2-switch-mode.sh

```diff
@@ -11,26 +11,32 @@
 set -e
 
 mode=${1:-query}
-device=${2:-"$(mstconfig q | grep "Device:" | awk 'BEGIN {FS="/";} { print $6 }')"}
+devices=${2:-"$(mstconfig q | grep "Device:" | awk 'BEGIN {FS="/";} { print $6 }')"}
 
-if [ -z "$device" ]; then
-    echo "Can't find device."
-    exit 120
-else
-    echo Found device: "${device}"
-fi
+for device in $devices; do
+    echo "Found device: $device"
+    type=$(mstconfig -d $device q | grep "Device type:" | awk '{ print $3}')
 
-type=$(mstconfig -d $device q | grep "Device type:" | awk '{ print $3}')
+    if [ "${type}" != "BlueField2" ]; then
```

Comment:
In anticipation for the BF3. Should we just do a regex here?

Reply:
Let's not anticipate too much. Just keep it hard-coded for Bluefield 2 for now.

Reply:
Sounds good. Is it possible to get lgtm/approved on this PR? Tested and verified it works on a cluster with multiple nics